Signal/src/call/CallService.swift

```diff
@@ -1028,21 +1060,27 @@ protocol CallServiceObserver: class {
 
         Logger.debug("\(TAG) in \(#function)")
 
-        PeerConnectionClient.stopAudioSession()
-        peerConnectionClient?.terminate()
-
-        peerConnectionClient = nil
+        self.localVideoTrack = nil
+        self.remoteVideoTrack = nil
+        self.fireDidUpdateVideoTracks()
         localVideoTrack = nil
         remoteVideoTrack = nil
         isRemoteVideoEnabled = false
+
+        var peerConnectionClientCopy = peerConnectionClient
```

Comment:
I've added a comment. This "copy" dance won't have any effect, but it's about future-proofing this code and being extra-conservative since WebRTC classes are so fragile and prone to throwing exceptions on teardown.